lib/ansible/cli/doc.py

```diff
@@ -1502,13 +1496,7 @@ def get_man_text(doc, collection_name='', plugin_type=''):
         if doc.get('plainexamples', False):
             text.append('')
             text.append(_format("EXAMPLES:", 'bold'))
-            if isinstance(doc['plainexamples'], string_types):
-                text.append(doc.pop('plainexamples').strip())
-            else:
-                try:
-                    text.append(DocCLI._dump_yaml(doc.pop('plainexamples')))
-                except Exception as e:
-                    raise AnsibleParserError("Unable to parse examples section", orig_exc=e)
+            text.append(doc.pop('plainexamples').strip())
```

Comment:
Since this is removing a feature, you need a changelog fragment announcing that. (And shouldn't this have a deprecation period?)